### DISCUSSION TITLE
Where exactly is my syntax error on github actions workflow file?

### DISCUSSION THREAD

**Discussion by joaocasarin** [community/community#26796] [Actions]
<p>Well, I’m trying to set up my github actions, but it keeps telling there’s a syntax error:</p>
<p>Link to the action: <a href="https://github.com/joaocasarin/test-selfapi/actions/runs/1364850588" class="inline-onebox" rel="noopener nofollow ugc">fix again · joaocasarin/test-selfapi@c992047 · GitHub</a><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/3/4351003a3ebee4891d1526ad1ba1cac53b5db3d2.png" data-download-href="/uploads/short-url/9Bvx7aFAUgGZikiTtLYvaa5rLoe.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/48847394/181099944-644b8475-aaf9-4370-a47a-67490c1cf1b6.png" alt="image" data-base62-sha1="9Bvx7aFAUgGZikiTtLYvaa5rLoe" width="690" height="340" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/4351003a3ebee4891d1526ad1ba1cac53b5db3d2_2_690x340.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/4351003a3ebee4891d1526ad1ba1cac53b5db3d2_2_1035x510.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/4351003a3ebee4891d1526ad1ba1cac53b5db3d2_2_1380x680.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/4351003a3ebee4891d1526ad1ba1cac53b5db3d2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1401×691 22.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>See the <a href="https://yaml.org/spec/1.2.2/#61-indentation-spaces">YAML spec, chapter 6.1</a>:</p>
<blockquote>
<p>To maintain portability, <a href="https://yaml.org/spec/1.2.2/#white-space-characters">tab</a> characters must not be used in indentation, since different systems treat <a href="https://yaml.org/spec/1.2.2/#white-space-characters">tabs</a> differently.</p>
</blockquote>
